FWCore/Services/test/full-site-local-config.testfile

```diff
@@ -1,8 +1,12 @@
 <site-local-config>
 <site name="DUMMY">
+<subsite name="DUMMY_SUB_SITE">
```

Comment:
Is this element missing a corresponding closing `</subsite>`?

Reply:
Yes, it was missing. Fixed now.